docs/remote_inference_blueprints/amazon_comprehend_connector_blueprint.md

```diff
@@ -0,0 +1,668 @@
+# Amazon Comprehend connector blueprint example for metadata embedding:
+
+Amazon Comprehend uses natural language processing (NLP) to extract insights about the content of documents without the need of any special preprocessing. You can get details of Amazon Comprehend from [Amazon Comprehend Documentation](https://docs.aws.amazon.com/comprehend/).
+
+## Language detection with [DetectDominantLanguage](https://docs.aws.amazon.com/comprehend/latest/APIReference/API_DetectDominantLanguage.html) API
+
+This instruction shows how to create OpenSearch connector having capability to set detected language code based on input text using Amazon Comrehend DetectDominantLanguage API.
+
+Note: Need to do this on 2.14.0 or later.
+
+### 1. Add connector endpoint to trusted URLs:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://comprehend\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
+
+### 2. Create connector for Amazon Bedrock:
+
+If you are using self-managed Opensearch, you should supply AWS credentials:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "Amazon Comprehend",
+  "description": "Test connector for Amazon Comprehend",
+  "version": 1,
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+    "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+    "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+  },
+  "parameters": {
+    "service_name": "comprehend",
+    "region": "ap-northeast-1",
+    "endpoint": "https://${parameters.service_name}.${parameters.region}.amazonaws.com",
+    "api": "Comprehend_20171127.DetectDominantLanguage",
```

Comment:
`Comprehend_20171127` Seems contains some date as version, is this the latest version? Where user can find the latest version in future?

Reply:
AWS recommends to lock API version for production to avoid disruption by new API version released[1]. Users can find the latest API version from SDK reference [2] or botocore code [3].

[1] https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/locking-api-versions.html
[2]  https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html
[3] https://github.com/boto/botocore/tree/master/botocore/data

I'm going to add a guide how to find latest API version

Reply:
I've added suggestion to refer demo notebook and tutorial.